src/main/java/no/nav/veilarbperson/client/kodeverk/KodeverkClientImpl.java

```diff
@@ -66,14 +70,27 @@ private Map<String, String> parseKodeverkBetydningJson(String responseJson) {
         JsonNode betydninger = rootNode.get("betydninger");
 
         betydninger.fieldNames().forEachRemaining((betydningName) -> {
-            JsonNode betydningNode = betydninger.get(betydningName).get(0);
+            JsonNode betydningerValues = betydninger.get(betydningName);
+            AtomicReference<JsonNode> betydningNyeste = new AtomicReference<>(betydninger.get(betydningName).get(0));
+
+            //find most recent value
+            if (betydningerValues.isArray()) {
+                ArrayNode arrayField = (ArrayNode) betydningerValues;
+                arrayField.forEach(node -> {
```

Comment:
En mulighet for å unngå variabelen `betydningNyeste` kunne vært å gjøre konvertere arrayField til en stream:
`StreamSupport.stream(arrayField.spliterator(), false).max((o1, o2) -> ...);`

Reply:
Fikset, bare må teste litt...